ihmc-interfaces/src/main/messages/ihmc_interfaces/behavior_msgs/msg/ContinuousHikingCommandMessage.msg

```diff
@@ -0,0 +1,42 @@
+# This message sends the current request for the Continuous Hiking process.
+# This holds parameters on how the Continuous Hiking process will behave.
+
+# Flag to start Continuous Hiking
+bool enable_continuous_hiking
+
+# Flag that allows the user to set a number of steps that will be taken, and Continuous Hiking will stop after that. This way the robot doesn't walk forever
+uint32 steps_before_safety_stop
+
+# Flag to walk straight forward with Continuous Hiking. This is generally a keyboard input, and is different then trying to walk with the Joystick
+bool walk_forwards
+
+# Flag to square up to the goal when finishing walking. We may want the robot to end at an exact location.
+# Setting this to true tells the robot to step on the final goal positions.
+bool square_up_to_goal
+
+# Flag to enable/disable planning with A* planner. This is the default planner to use
+bool use_astar_footstep_planner
+
+# Flag to enable/disable planning with monte-carlo footstep planner
+bool use_monte_carlo_footstep_planner
```

Comment:
Should the planner selection be a string or a short or something? Like, it wouldn't make sense to enable both astar and monte carlo.

Reply:
Maybe in the future that might make sense. I haven't put much thought into it. On one of my branches you can do this "hybrid" plan where if both of those are true. Then you do the Monte Carlo Planner and you use that plan as a reference to the A* Planner. That could probably be handled with a string or short,  I'm not trying to change that implementation to much at the moment. Much lower on the priority then some other tasks.

Reply:
So for now I'd like to leave this, and if it causes problems going forward I can change it then.